metatron-app/metatron-discovery
# Fn fix custom color setting

### Description
Displayed multiple tooltip on custom color setting input box

**Related Issue** : 
통합테스트

### How Has This Been Tested?
1. chart
2. put some dimension and measure on the layer
3. Click color option config on lnb menu
4. enable custom color setting
5. change color value and check tool tip display
- Custom color tooltips, tooltips are displayed when mouse over in the focused state.


### Types of changes



### Checklist:
